ios/SourceModule.swift

```diff
@@ -66,8 +90,13 @@ class SourceModule: NSObject, RCTBridgeModule {
      - Parameter drmNativeId: ID of the DRM config object to use.
      - Parameter config: `SourceConfig` object received from JS.
      */
-    @objc(initWithConfig:drmNativeId:config:)
-    func initWithConfig(_ nativeId: NativeId, drmNativeId: NativeId?, config: Any?) {
+    @objc(initWithConfig:drmNativeId:config:sourceRemotePlaybackConfig:)
+    func initWithConfig(
+        _ nativeId: NativeId,
+        drmNativeId: NativeId?,
+        config: Any?,
+        sourceRemotePlaybackConfig: Any
```

Comment:
Should this maybe be `Any?` as it is optional?

Reply:
fixed in ee6f953

(Fun fact: you can store anything in `Any`, even optionals, so this would never fail. But looks better with `Any?` :) )